documentation/system-design/spec/generator/database.ts

```diff
@@ -0,0 +1,30 @@
+const filename = "database.pdf";
+import * as path from "https://deno.land/std@0.120.0/path/mod.ts";
+
+export async function renderDatabase() {
+  const p = Deno.run({
+    cmd: ["bash"],
+    stdout: "piped",
+    stdin: "piped",
+  });
+
+  const encoder = new TextEncoder();
+
+  const command = `npx -q @softwaretechnik/dbml-renderer -i ${path.join(
+    "..",
+    "..",
+    "..",
+    "..",
+    "Book-Bazar",
```

Comment:
I think it might be a good idea to remove these lines. If you have the original repo naming, the database won't generate because of this.

Reply:
oh lol, good idea